docs/source/en/serialization.mdx

```diff
@@ -74,7 +74,6 @@ Ready-made configurations include the following architectures:
 - GPT-J
 - GroupViT
 - I-BERT
-- ImageGPT
```

Comment:
Is it intended to remove `ImageGPT`?

Reply:
sorry, I think I merged older version of main. It's fixed.😅